stdlib/event/event.lua

```diff
@@ -4,8 +4,25 @@
 require 'stdlib/core'
 require 'stdlib/game'
 
+core_events = { init = -1, load = -2, configuration_changed= -3 }
```

Comment:
I'd move core_events to the Event object on line 9 below, so we aren't polluting the space with too many globals. Maybe call it Event.core_events.init or even Event.core.init... One of the two anyway.